chord_metadata_service/chord/ingest/experiments.py

```diff
@@ -146,6 +148,11 @@ def ingest_experiments_workflow(json_data, dataset_id: str) -> list[em.Experimen
 
     exps = json_data.get("experiments", [])
 
+    if len(exps) == 0:
+        # If empty experiments array
+        # Validate an empty json to raise an IngestError with validation details
+        validate_experiment({})
```

Comment:
instead of this, could we build a super-schema for the ingest structure which stipulates a minimum array length of 1?

Reply:
looking into it